register.go

```diff
@@ -127,33 +239,51 @@ func unregister() error {
 		return fmt.Errorf("warning: the system is already unregistered")
 	}
 
-	if err := conn.Object("com.redhat.RHSM1", "/com/redhat/RHSM1/Unregister").Call("com.redhat.RHSM1.Unregister.Unregister", dbus.Flags(0), map[string]string{}, "").Err; err != nil {
-		return unpackError(err)
+	if err := conn.Object(
```

Comment:
This is purely a formatting difference, but it might make these longer function calls more legible. Rather than putting the err declaration inline within the `if` condition, move it to an explicit line:

```go
err := conn.Object(
    "com.redhat.RHSM1",
	"/com/redhat/RHSM1/Unregister").Call(
	"com.redhat.RHSM1.Unregister.Unregister",
	dbus.Flags(0),
	map[string]string{},
	"")
if err != nil {
    return unpackRHSMErrorMsg(err)
}
```

Reply:
OK, np.